distribution/ddf-common/src/main/resources/etc/system.properties

```diff
@@ -31,6 +31,9 @@ javax.net.ssl.trustStorePassword=changeit
 javax.net.ssl.keyStoreType=jks
 javax.net.ssl.trustStoreType=jks
 
+#force java to run in headless mode for when the server doesn't have a display device
+java.awt.headless=true
```

Comment:
Correct everything runs fine headless I was able to run on my on machine successfully and on a docker instance that did not have a display device and successfully ingest a jpg that generated a thumbnail.